backend/main.py

```
@@ -1,78 +1,200 @@
+        memory=(512, 1024),
+        cpu=2,
+        allow_concurrent_inputs=10,
+        timeout=60 * 10,
+    )
+    @asgi_app()
+    def api():
+        return app
+
+    @modal_app.function(image=image, schedule=Cron('* * * * *'))
+    async def notifications_cronjob():
+        await start_cron_job()
```

Ruff

Undefined name `start_cron_job`

(F821)